RedBanana/src/RedBanana/Bytecode.hs

```diff
@@ -202,4 +212,9 @@ disasm (i:is) = instr:rest
                         disasm is
 
 disasmFile :: String -> IO [Instr]
-disasmFile filename = fmap (disasm . hexToBytes) $ readFile filename
\ No newline at end of file
+disasmFile filename = fmap (disasm . hexToBytes) $ readFile filename
+
+printOpCodes :: String -> IO ()
+printOpCodes filename = do
```

Comment:
Could also be written as 
```haskell
printOpCodes filename = mapM_ print =<< disasmFile filename
```

Not full point-free tho ☺️

Reply:
Cool I was looking for this yesterday but was too lazy to find it :)
Also replaced the fmap with <$> which looks better